auth_plugins/rollcontrol/plugin.py

```diff
@@ -14,6 +16,11 @@
 class RollPlugin(BotAuthPlugin):
     def __init__(self, load_from_disk=True):
         super().__init__()
+        if os.getenv("PROMETHEUS_PORT"):
```

Comment:
You should validate the `PROMETHUES_PORT` is an actual integer before casting it:-

```
if os.getenv("PROMETHEUS_PORT") and os.getenv("PROMETHUES_PORT").isdigit():
```